view/components/header.html

```diff
@@ -1,3 +1,3 @@
 <div id="inner-component-header">
-    <strong>Jarvis</strong> <span id="PA">personal assistant</span>
+    <h4 style="color:floralwhite;margin-left: 1%;padding: 0;">Jarvis : <span id="PA" style="font-family:cursive;">The personal assistant</span></h4> 
```

Comment:
please don't hardcode the stylings. maintain defined standards, by either making these styles by defining them in a class or specify the id.